editor/src/clj/editor/collision_object.clj

```diff
@@ -409,12 +411,61 @@
                   (outline/gen-node-outline-keys (map (comp shape-type-label :shape-type)
                                                       shapes)))))))
 
+(defn convex-hull-scene
+  [_node-id convex-shape-data color]
+  (when (and (= (:shape-type convex-shape-data) :type-hull)
+             (not-empty (:data convex-shape-data)))
```

Comment:
Trying to be defensive here, since this file is produced by an external tool. I guess there are too many things that can go wrong here that we could check. Are the points valid? Is the shape actually convex? I am also not sure if we are supposed to support any other `shape-type`.

Reply:
Not sure if Vlad mentinoed that we want to support more than one shape at the same time?
While the current workflow is "single file -> single hull", we're likely to introduce the fact that we can have multiple hulls in the same file.

We're not there yet, but I figured I should mention it while we have it fresh in memory, in case you'd like to make any adjustments in your PR (e.g. not assuming it's a single hull)

Reply:
@JCash Thanks for the input! I tried adding another shape to the file, and its points seem to be part of `:data` already, so they are going to be rendered.